kotlin/android/app/build.gradle

```diff
@@ -32,10 +32,10 @@ android {
 
             // Debug Config
             manifestPlaceholders.hostName = "app.firez.one"
-            buildConfigField("String", "AUTH_HOST", "\"localhost\"")
+            buildConfigField("String", "AUTH_HOST", "\"10.0.2.2\"")
```

Comment:
This is the host's IP as seen by the android debugging environment.

Reply:
Nevermind, using `app.firez.one` since testing locally with Docker is problematic